cirq/protocols/resolve_parameters.py

```diff
@@ -56,7 +56,9 @@ def is_parameterized(val: Any) -> bool:
         return False
 
 
-def resolve_parameters(val: Any, param_resolver: 'cirq.ParamResolver') -> Any:
+def resolve_parameters(
+        val: Any,
+        param_resolver: 'cirq.ParamResolverOrSimilarType') -> Any:
```

Comment:
does this have to be quoted?  Can't you "from cirq.study import resolvers" and use resolvers.PROST?  This feels a bit better to me because my IDE treat it as a type I can jump to, whereas for a string it doesn't do this.

Reply:
Yes, it has to be quoted. It breaks a dependency cycle.

When I ctrl+click on the string in pycharm, it takes me to the definition. At least some IDEs understand that string type annotations refer to code.